src/jobs_queue.erl

```diff
@@ -259,4 +259,4 @@ empty(#queue{st = #st{table = T}} = Q) ->
 
 is_expired(TS, Now, TO) ->
     MS = Now - TS,
-    MS > TO.
+    MS > TO * 1000.
```

Comment:
Yes, but I think it would be better to convert to microsecs before starting the iteration, i.e. on line 241